src/sass/style.scss

```diff
@@ -97,6 +96,36 @@ button:hover {
 	display: block;
 }
 
+.replyButton {
+	float: right;
+}
+
+#replyButton{
+	background-color: Transparent;
+	outline: none;
+	border-width: 0px;
+}
+
+.replyTweetModal {
```

Comment:
do you need top right and button?

Reply:
it was to ensure it expanses vertically but to go all the way to the right, did you want me to do this differently?

Reply:
if that works it's fine
you could probably use height 100% as well but minor